mcux/drivers/imxrt6xx/CMakeLists.txt

```diff
@@ -25,3 +25,5 @@ zephyr_sources_ifdef(CONFIG_COUNTER_MCUX_LPC_RTC	fsl_rtc.c)
 if(CONFIG_FLASH_MCUX_FLEXSPI_XIP)
   zephyr_code_relocate(fsl_flexspi.c ${CONFIG_FLASH_MCUX_FLEXSPI_XIP_MEM}_TEXT)
 endif()
+zephyr_library_sources_ifdef(CONFIG_ENTROPY_MCUX_TRNG	fsl_trng.c)
+
```

Comment:
Make this `zephyr_sources_ifdef` to be consistent with the rest of the file, and don't add the extra newline.

Reply:
done